src/Coalesce.Starter.Web/Styles/site.scss

```diff
@@ -54,22 +54,6 @@ input[type="checkbox"].input-validation-error {
     display: none;
 }
 
-/* Set height of Select 2 to make it the same as bootstrap  */
-span.select2-container span.select2-selection--single {
-    height: 34px;
-}
-
-span.select2-container--default .select2-selection--single .select2-selection__rendered {
-    line-height: 33px;
-}
-
-span.select2-container--default .select2-selection--single .select2-selection__arrow {
-    height: 30px;
-}
-
-.select2-container {
```

Comment:
I think this rule for the width may be separate from the other select2 rules here. The goal behind this was to replicate Bootstrap's behavior of `.form-control { width: 100% }`. Looking at other projects, I think a more refined version of this rule came along at some point in the form of `span.select2.select2-container { width: 100% !important; }`. 

If the Bootstrap theme for select2 already does this, then that's great too. Its not that important that we keep this since its not like we're making breaking changes to existing projects, but I did want to set down in writing what I'm hazily recollecting as the original intent behind this particular rule.

Reply:
perfect!